external/Apr1Lib.cmake

```diff
@@ -0,0 +1,160 @@
+# -----------------------------------------------------------------------------
+# Numenta Platform for Intelligent Computing (NuPIC)
+# Copyright (C) 2016, Numenta, Inc.  Unless you have purchased from
+# Numenta, Inc. a separate commercial license for this software code, the
+# following terms and conditions apply:
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU Affero Public License for more details.
+#
+# You should have received a copy of the GNU Affero Public License
+# along with this program.  If not, see http://www.gnu.org/licenses.
+#
+# http://numenta.org/licenses/
+# -----------------------------------------------------------------------------
+
+# Creates ExternalProject for building the apr-1 static library
+# (apache public runtime)
+#
+# Exports:
+#   LIB_STATIC_APR1_INC_DIR: directory of installed apr-1 lib headers
+#   LIB_STATIC_APR1_LOC: path to installed static apr-1 lib
+
+get_filename_component(REPOSITORY_DIR ${PROJECT_SOURCE_DIR}/.. ABSOLUTE)
+
+set(aprlib_install_prefix "${EP_BASE}/Install/Apr1StaticLib")
+set(aprlib_install_lib_dir "${aprlib_install_prefix}/lib")
+
+# Export directory of installed apr-1 lib headers to parent
```

Comment:
These comments saying they are exported to the parent are confusing. There is a `PARENT_SCOPE` option for the `set` command that isn't used here (and isn't necessary depending on how CMakeLists.txt includes this). So perhaps reword / drop the parent comment?